### ISSUE TITLE
Make smarter clickable executables (.bat and .sh)

### ISSUE THREAD

**Issue by grrttedwards** [grrttedwards/wow-addon-updater#14]

**Comment by enpinzolas** [comment]
Hey, thank you for updating this great utility!

For this issue, making the clickables work with pipenv is quite simple, although I'm not sure if that was your aim.

For windows, something as simple as this works.
```bat
pipenv run python -m updater
PAUSE
```

And for Unix, although untested, this should sufice:

```sh
pipenv run python -m updater
read -n1 -r -p "Press anything to close..."
```

**Comment by grrttedwards** [comment]
@enpinzolas Hey! Thanks for stopping by. 

For now, I think what you suggest would be good for people who are capable of running a few commands on the command line and just want a quick way to execute it.

I'm actually looking for something perhaps a little more complex -- like checking if you have pip, then pipenv, then installing the Pipfile if needed... and just giving a little bit nicer message if the prerequisites are not there.

**Comment by enpinzolas** [comment]
Oh, it can be done in bash at least, but I guess that if it's that complex, it would be best to make a helper python script (since python **should** be installed beforehand) that does all that meddling, and then execute the program itself.

I'll make a pull request once I get something valid if you don't mind.

**Comment by grrttedwards** [comment]
Feel free! The only thing about that is that not everyone (including myself) has python as the default to execute a `.py` file. Then we're kind of back at square one. Unless you mean the helper script does all the work, but there are still script files for launching `python install_wrapper.py`

**Comment by enpinzolas** [comment]
Yeah, I meant using a python script as a side step before executing `pipenv run python -m updater` on the main script.
